---
int16[360] hist